spec/services/idv/proofing_components_spec.rb

```diff
@@ -195,16 +196,6 @@
         expect(subject.resolution_check).to eql('AReallyGoodVendor')
       end
     end
-
-    context 'when resolution done but resolution_vendor nil because of 50/50 state' do
```

Comment:
I see a test case above on line 172 about "50/50 state" with `residential_resolution_vendor`. Do we still need that?

Reply:
Nope!